pkg/discovery/endpoint/restapi/operations.go

```diff
@@ -19,25 +21,28 @@ var logger = log.New("discovery-rest")
 // API endpoints.
 const (
 	wellKnownEndpoint = "/.well-known/did-orb"
+	webFingerEndpoint = "/.well-known/webfinger"
 )
 
-type wellKnowResp struct {
-	ResolutionEndpoint string `json:"resolutionEndpoint"`
-	OperationEndpoint  string `json:"operationEndpoint"`
-}
+const (
+	minResolvers      = "https://trustbloc.dev/ns/min-resolvers"
+	minResolversValue = 2 // TODO need to be configurable
```

Comment:
Need to also handle cases where there isn't a shared domain (aka min resolvers can only be 1 for those cases) - likely the default.